src/parcsr_ls/par_relax_device.c

```diff
@@ -117,7 +117,7 @@ hypre_BoomerAMGRelaxTwoStageGaussSeidelDevice ( hypre_ParCSRMatrix *A,
    for (i = 0; i < num_inner_iters; ++i)
    {
       // 2) r = Lz
-      hypre_CSRMatrixSpMVDevice(1.0, A_diag, z_local, 0.0, r_local, -2);
+      hypre_CSRMatrixSpMVDevice(0, 1.0, A_diag, z_local, 0.0, r_local, NULL, -2);
```

Comment:
I'm just curious... why do we directly call the custom SpMV here instead of calling `hypre_CSRMatrixMatvecDevice` and letting it choose a default implementation?

Reply:
We want to multiply the lower triangular part of a matrix (-2: strict lower), which is not supported by general MatvecDevice (using cusparse e.g.)